src/motion/features/layout/Animate.tsx

```diff
@@ -72,6 +74,15 @@ class Animate extends React.Component<AnimateProps> {
         const boxHasMoved = hasMoved(origin, target)
 
         const animations = eachAxis(axis => {
+            /**
+             * If layout is set to "position", we can resize the origin box based on the target
+             * box and only animate its position.
+             */
+            if (layout === "position") {
+                const targetLength = target[axis].max - target[axis].min
+                origin[axis].max = origin[axis].min + targetLength
+            }
```

Comment:
Just so I'm clear, we're adding to the max so that the later calculation of the width/height difference doesn't create a flip'd scale, right?

Reply:
Essentially yeah, it'll be FLIPped in terms of position but not scale.